app/src/main/res/menu/menu_search_engine_manual_add.xml

```diff
@@ -5,6 +5,14 @@
 
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <!-- We modify the Setting's theme's actionButtonStyle in order to change the padding between
+         these two items - not great, but it works. Details in @style/SettingsActionButtonStyle. -->
+    <item android:id="@+id/learn_more"
+          android:icon="@drawable/ic_menu_info"
+          android:title="@string/preference_autocomplete_learn_more"
+          app:showAsAction="always"/>
+
```

Comment:
Mhhh. Having this icon and the "Save" string in the toolbar looks pretty weird on my phone tbh.

Reply:
All other settings include a "Learn More" link at the bottom - why did we decide to add something to the toolbar?

Reply:
tbh, I'm not sure and I think it's too late to fix this (given the need to uplift). We could file a bug to further investigate the UX concerns you have.